src/Esprima/Scanner.cs

```diff
@@ -947,12 +955,34 @@ public Token ScanHexLiteral(int start)
             };
         }
 
+        private Token ScanBigIntLiteral(int start, ReadOnlySpan<char> number, NumberStyles style)
+        {
+            var c = number[0];
+            if (c >= 8 && c <= 'F')
+            {
+                // ensure we get positive number
+                number = ("0" + number.ToString()).AsSpan();
+            }
+#if !NETFRAMEWORK && !NETSTANDARD2_0
```

Comment:
```suggestion
#if NETSTANDARD2_1_OR_GREATER
```

Is that what you meant?

Reply:
Does it include everything like NETCOREAPP31 onwards? standard vs modern net5/6 can be confusing.